src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -106,14 +118,16 @@ def reset(self) -> None:
         self.observations = []
         # Create an instance memory to associate bounding boxes in space
         self.instances = InstanceMemory(
-            num_envs=1, du_scale=1, instance_association="bbox_iou"
+            num_envs=1,
+            **self.instance_memory_kwargs,
```

Comment:
Should we have these default args set somewhere?

Reply:
Good idea (though num_envs will always be 1 here)